pkg/reporter/sbom/models.go

```diff
@@ -0,0 +1,34 @@
+package sbom
+
+import (
+	"github.com/google/osv-scanner/pkg/models"
+
+	"github.com/CycloneDX/cyclonedx-go"
+)
+
+type CycloneDXVersion int
+
+const (
+	CycloneDXVersion14 CycloneDXVersion = iota
+	CycloneDXVersion15
+)
+
+var SpecVersionToBomCreator = map[CycloneDXVersion]BomCreator{
```

Comment:
We will wish to add SPDX support in the future as well, can you rename this to CycloneDXBomCreator?